themes/gatsby-starter-blog-theme-core/src/gatsby-theme-blog/gatsby-plugin-theme-ui/colors.js

```diff
@@ -0,0 +1,25 @@
+import merge from "deepmerge"
```

Comment:
I might be misunderstanding the purpose of `gatsby-starter-blog-theme-core`, but is this intended for latent shadowing? I'd imaging that the starter for blog-core would be fairly minimal and decoupled from `gatsby-theme-blog`

Reply:
I think I copied this starter and left the directory here. will delete.